osquery/tables/forensic/carves.cpp

```diff
@@ -0,0 +1,122 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <boost/algorithm/string/join.hpp>
+#include <boost/property_tree/ptree.hpp>
+#include <boost/uuid/uuid.hpp>
+#include <boost/uuid/uuid_generators.hpp>
+#include <boost/uuid/uuid_io.hpp>
+
+#include <osquery/core.h>
+#include <osquery/database.h>
+#include <osquery/dispatcher.h>
+#include <osquery/flags.h>
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+
+#include "osquery/carver/carver.h"
+#include "osquery/core/json.h"
+
+namespace pt = boost::property_tree;
+
+namespace osquery {
+
+DECLARE_bool(disable_carver);
+
+namespace tables {
+
+/// Database prefix used to directly access and manipulate our carver entries
+const std::string kCarverDBPrefix = "carves.";
+
+void enumerateCarves(QueryData& results) {
+  std::vector<std::string> carves;
+  auto s = scanDatabaseKeys(kQueries, carves, kCarverDBPrefix);
```

Comment:
I'm just going to remove it. If we fail to grab carves from the DB, the vector _should_ just be empty so our `for` loop will fall through. Thanks!